<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>23</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="dbTypeLabel">
      <property name="text">
       <string>dbType:</string>
      </property>
      <property name="buddy">
       <cstring>dbTypeComboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="dbTypeComboBox">
      <item>
       <property name="text">
        <string>Local</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Server</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="dbLabel">
      <property name="text">
       <string>database:</string>
      </property>
      <property name="buddy">
       <cstring>dblineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="dblineEdit">
      <property name="text">
       <string>testdb</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="usernameLabel">
      <property name="text">
       <string>User Name:</string>
      </property>
      <property name="buddy">
       <cstring>usernameLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="usernameLineEdit">
      <property name="text">
       <string>root</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="pwdLable">
      <property name="text">
       <string>password:</string>
      </property>
      <property name="buddy">
       <cstring>pwdLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="pwdLineEdit">
      <property name="text">
       <string>yp*963.</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
